scripts/generate_example_previews.py

```
@@ -0,0 +1,135 @@
+        browser = launch_certification_browser(playwright)
+        context = browser.new_context(viewport=VIEWPORT, device_scale_factor=2)
+        page = context.new_page()
+        for slug in EXAMPLE_SLUGS:
+            url = f"{base_url}/site-dist/examples/{slug}/index.html"
+            response = None
+            # "networkidle" is flaky by nature (Playwright's own docs warn
+            # against relying on it) and reliably reproduced spurious 404s
+            # here after a few pages' worth of prior navigations -- "load"
+            # is deterministic for these static, no-polling pages. The
+            # retry loop stays as a second line of defense.
+            for attempt in range(3):
```

Ruff (0.16.1)

[warning] Loop control variable `attempt` not used within loop body

Rename unused `attempt` to `_attempt`

(B007)